programs/ssl/ssl_client2.c

```diff
@@ -2234,6 +2317,44 @@ int main( int argc, char *argv[] )
     }
 #endif
 
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+    if( opt.use_srtp != DFL_USE_SRTP )
```

Comment:
Minor: This forces the default value to be `0`/off - it wouldn't work anymore if it changed to `1`/on.

Suggestion: Modify the code to be agnostic of the default values for command line parameters.

Reply:
This is a defect with pre-existing command line parameters, but that shouldn't prevent us from doing it right when introducing new ones.

Reply:
fixed